app/js/components/Draft.jsx

```diff
@@ -2,17 +2,50 @@ import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import shortid from 'shortid';
+import constants from '../utils/constants';
+import IconButton from './button/IconButton';
 
 export class Draft extends PureComponent {
   renderDraftList = () => {
     const { draftOrders } = this.props;
     return draftOrders.map((order) => {
+      const isPanel = !!order.set;
+      const draftType = !isPanel ? 'single' : 'panel';
       const orderName = order.display || order.drugName;
+      const iconClass = classNames(
+        'scale',
+        {
+          'i-gray': order.urgency === constants.ROUTINE || order.urgency === undefined,
```

Comment:
Please use `if (typeof myVar === "undefined")` (https://stackoverflow.com/questions/27509/detecting-an-undefined-object-property)

Reply:
Thanks for the feedback @jomadoye, unary corrected.